lib/widgets/add_to_wallet_button.dart

```diff
@@ -1,30 +1,58 @@
+import 'dart:async';
+import 'dart:math';
+
+import 'package:add_to_wallet/add_to_wallet.dart';
 import 'package:flutter/foundation.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter/services.dart';
 
-class AddToWalletButton extends StatelessWidget {
+class AddToWalletButton extends StatefulWidget {
+  static const viewType = 'PKAddPassButton';
+
   final List<int> pkPass;
   final double width;
   final double height;
   final Widget? unsupportedPlatformChild;
+  final FutureOr<void> Function()? onPress;
+  final String _id = Random.secure().nextDouble().toString();
```

Comment:
Generating a pseudo random string to identify the widget. We are not supposed to have a screen with more than a AddToWallet button  so it should not be a problem that uniqueness is not guaranteed here.

Reply:
Agree. Just thinking in release this as public, we can think in a screen that have a list of items which each item can be added to wallet. We can just generate a UUIDv4 for that, but I think that is ok meantime.

Reply:
I switch to Uuid as it's not too much effort